### DISCUSSION TITLE
Socket is not connected on https request in React.js

### DISCUSSION THREAD

**Discussion by MuhammadJibranBaig** [socketio/socket.io#3852] [Q&A]
Backend is on Golang ,when I used to connect socket on http request it works but when I connect socket on https request it didn't work.

![notconnected](https://user-images.githubusercontent.com/64801996/111941413-cfebfc00-8af2-11eb-8eb9-af19eecf7b29.PNG)
![err](https://user-images.githubusercontent.com/64801996/111941420-d5494680-8af2-11eb-9ff6-069113154048.PNG)

**Comment by darrachequesne** [comment]
> Backend is on Golang

Which server implementation are you using? Is it https://github.com/googollee/go-socket.io? Could you please check the compatibility between the server and the client?
